overlord/snapstate/aliasesv2.go

```diff
@@ -201,6 +201,11 @@ func autoAliasesDelta(st *state.State, names []string) (changed map[string][]str
 	return changed, dropped, firstErr
 }
 
+func nonDaemon(info *snap.Info, appName string) bool {
+	app := info.Apps[appName]
+	return app != nil && app.Daemon == ""
```

Comment:
interesting question, I don't see changes to 

https://github.com/snapcore/snapd/blob/master/wrappers/binaries.go#L37

in #2592 

@mvo?

if we need to not have binaires for those maybe we can move to have a helper wrappers.HasBinary(app)  , sounds like something to do in #2592 though